_schemaV2.graphql

```diff
@@ -701,7 +701,6 @@ type Artist implements Node & Searchable & EntityWithFilterArtworksConnectionInt
   isFollowed: Boolean
   isPublic: Boolean
   isShareable: Boolean
-  isTargetSupply: Boolean
```

Comment:
@damassi This doesn't seem right, but I don't know why it's getting removed when i run `dump:local` to regenerate the schemas.